docs/UserGuide.md

```diff
@@ -651,6 +649,20 @@ Exiting program, goodbye!
 Explanation:
 This command will exit the program. Our program will print a statement to bid you goodbye!
 
+### Example 2
+
+Input:
+```
+exit with some extra arguments
+```
```

Comment:
Perhaps can give some examples of what you mean by ‘extra arguments’

Reply:
`exit with some extra arguments` was meant to be a literal string that the user inputs.

I will explain in the explanation to clarify it.